python/hsfs/feature_group.py

```diff
@@ -2743,15 +2748,17 @@ def read(
         # Raises
             `hsfs.client.exceptions.RestAPIError`.
         """
-        if engine.get_type() == "python" and not online:
-            raise FeatureStoreException(
-                "Reading an External Feature Group directly into a Pandas Dataframe using "
-                + "Python/Pandas as Engine from the external storage system "
-                + "is not supported, however, if the feature group is online enabled, you can read "
-                + "from online storage or you can use the "
-                + "Query API to create Feature Views/Training Data containing External "
-                + "Feature Groups."
-            )
+
+        #
+        # if engine.get_type() == "python" and not online:
+        #    raise FeatureStoreException(
+        #        "Reading an External Feature Group directly into a Pandas Dataframe using "
+        #        + "Python/Pandas as Engine from the external storage system "
+        #        + "is not supported, however, if the feature group is online enabled, you can read "
+        #        + "from online storage or you can use the "
+        #        + "Query API to create Feature Views/Training Data containing External "
+        #         + "Feature Groups."
+        #    )
```

Comment:
```suggestion
```

Reply:
I make it print the regular error message if the external fg not supported by flyingduck or if flyingduck is not enabled/reachable